### ISSUE TITLE
Support unwired labs as a fallback provider

### ISSUE THREAD

**Issue by hannosch** [mozilla/ichnaea#508]
Our geolocation logic supports falling back to a different provider in case our own database does not find a good enough answer.

Currently we only support falling back to GLS/MLS/combain compatible providers. We should support other providers, notably unwiredlabs as well.

**Comment by hannosch** [comment]
@Manicben I've started making the fallback provider more pluggable, adding a fallback_schema column on each API key and looking up the schema based on this in the fallback provider. I haven't added the specific unwired labs schema yet. Would this be something you could help with?

**Comment by Manicben** [comment]
I've written the Unwired labs schema and hopefully it works, since I am unable to test it.
https://gist.github.com/Manicben/01c86faea78abcaa3eacb221d0df14ee

I assume some other changes to fallback.py will be needed, such as:
```python
UNWIREDLABS_SCHEMA = 'unwiredlabs/v1'

self.outbound_schemas = {
    DEFAULT_SCHEMA: ICHNAEA_V1_OUTBOUND_SCHEMA,
    UNWIREDLABS_SCHEMA: UNWIREDLABS_V1_OUTBOUND_SCHEMA,
}
self.result_schemas = {
    DEFAULT_SCHEMA: ICHNAEA_V1_RESULT_SCHEMA,
    UNWIREDLABS_SCHEMA: UNWIREDLABS_V1_RESULT_SCHEMA,
}
```

Not sure what else is needed, let me know.

**Comment by hannosch** [comment]
Thank you very much for those schemata. I've used those as a starting point and put some finishing touches on top of them.

I think my last commit does most of the work, but it still misses a way to read the external API token from the database and put it into the outgoing request body.

**Comment by hannosch** [comment]
Ok, I've finished with the other bits. The unwiredlabs token can now be specified in the fallback_url via appending ``#some_secret_token``. This is parsed and the anchor fragment is removed from the URL before using it.

In related news via 849ae8362210bdd5f174fc957b67b7393b5c1ef5 and 91db78afc6a6ba04c3a795f057b180ae357fc370 I've separated the fallback cache on a per fallback schema basis. So one could use both say combain and unwiredlabs in the same ichnaea install, possibly use caching for both and their cache contents would no longer get mixed.
